GetJobber/atlantis
# docs: Storybook in atlantis

## Motivations

We want to move Docz documentation to Storybook. This set us up to migrate documentation over.

Keep in mind that Docz still exists and will be the one you'll see on the preview deploy until we move all the existing one to Storybook.



## Changes



### Added

- Installed storybook for react
- Added mdx linters
- Added `.stories.mdx` for Button (kept the original .mdx file since Docz need that)

### Changed

- 

### Deprecated

- 

### Removed

- 

### Fixed

- 

### Security

- 

## Testing


- Pull down this branch
- run `npm ci`
- run `npm run storybook`
- You should see the storybook runs with Button in it
- run `npm start` to ensure Docz still runs

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

![Random photo of Atlantis](https://loremflickr.com/672/400/atlantis)

JOB-61681
